packages/store-engine/src/main/engine/FileEngine.ts

```diff
@@ -12,6 +12,7 @@ import {
 
 export default class FileEngine implements CRUDEngine {
   public storeName: string = '';
+  public storeNameOriginal: string = '';
```

Comment:
Instead of having both (`storeName` & `storeNameOriginal`) we shoul decide on the one to keep because I doubt that we really need to keep both.